tests/pkg/notifications/context-keys/keys.go

```diff
@@ -3,6 +3,10 @@ package context_keys
 const (
 	FormationIDKey           = "formationIDKey"
 	FormationNameKey         = "formationNameKey"
+	SecondFormationIDKey           = "secondFormationIDKey"
+	SecondFormationNameKey         = "secondFformationNameKey"
```

Comment:
1. the file can be formatted
2. typo in the value of the `SecondFormationNameKey` var -> should be `secondFormationNameKey`
3. `ApplicationID` seems to be unused so it can be deleted